packages/app/src/context/createDataContext.jsx

```diff
@@ -8,9 +8,9 @@ export default (reducer, actions, initialState) => {
 
     const boundActions = {};
 
-    for (let key in actions) {
-      boundActions[key] = actions[key](dispatch);
-    }
+    actions.forEach((element, index) => {
```

Comment:
Aqui as regras que dão error são: guard-for-in e no restricted-syntax. Nesse caso olhando as documentações do Eslint (https://eslint.org/docs/rules/guard-for-in e https://eslint.org/docs/rules/no-restricted-syntax) podemos melhorar mto o código em desabilitar nenhuma regra. Isso seria melhor:

` Object.keys(actions).forEach(key => {
      boundActions[key] = actions[key](dispatch);
    });`